From 8a5d3d24b2e9852717151901410029015ee75d00 Mon Sep 17 00:00:00 2001
From: Sai Krishna Potthuri <lakshmi.sai.krishna.potthuri@xilinx.com>
Date: Thu, 26 Dec 2019 04:16:24 -0700
Subject: [PATCH 0927/1851] mtd: spi-nor: Add entry for mt35xu01g flash

commit 78f12536a459268e5ab46ee8dca4be94a5fc4212 from
https://github.com/Xilinx/linux-xlnx.git

Add entry for mt35xu01g Micron NOR flash.

Signed-off-by: Sai Krishna Potthuri <lakshmi.sai.krishna.potthuri@xilinx.com>
Signed-off-by: Radhey Shyam Pandey <radhey.shyam.pandey@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/mtd/spi-nor/spi-nor.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/drivers/mtd/spi-nor/spi-nor.c b/drivers/mtd/spi-nor/spi-nor.c
index 60e4f03b80e1..33ff3db78873 100644
--- a/drivers/mtd/spi-nor/spi-nor.c
+++ b/drivers/mtd/spi-nor/spi-nor.c
@@ -2515,6 +2515,9 @@ static const struct flash_info spi_nor_ids[] = {
 			SECT_4K | USE_FSR | SPI_NOR_OCTAL_READ |
 			SPI_NOR_OCTAL_WRITE | SPI_NOR_4B_OPCODES)
 	},
+	{ "mt35xu01g",  INFO(0x2c5b1b, 0, 128 * 1024, 1024,
+			    SECT_4K | USE_FSR | SPI_NOR_OCTAL_READ |
+			    SPI_NOR_OCTAL_WRITE | SPI_NOR_4B_OPCODES) },
 	{ "mt35xu02g",  INFO(0x2c5b1c, 0, 128 * 1024, 2048,
 			     SECT_4K | USE_FSR | SPI_NOR_OCTAL_READ |
 			     SPI_NOR_OCTAL_WRITE | SPI_NOR_4B_OPCODES) },
-- 
2.31.1

